wren-ui/migrations/20240430033014_update_model_column_table.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
@onlyjackfrost could you add some comments here on why we're taking out this column? thx

Reply:
delivered in 9ec6f9620239b869a9716daf8c6827d85c23995a